### ISSUE TITLE
Velocity is not defined when process.env.NODE_ENV === 'test'

### ISSUE THREAD

**Issue by moretti** [twitter-fabric/velocity-react#127]
One of the recent PRs added the following check https://github.com/twitter-fabric/velocity-react/commit/6e6d505c9556089d587b48ca531708e9ab2117e9 which breaks tests running in real browsers.

When I run tests with `karma` + `karma-webpack`(and `NODE_ENV`=`test`) I get the following exception:

```
TypeError: Cannot read property 'removeData' of undefined
    at Constructor._clearVelocityCache (http://localhost:9876/base/test/index.js?0bc9e53dbd6efbdb25d8eea4591af0bf094c9d64:70135:26)
```

because `Velocity.Utilities` is now undefined:

https://github.com/twitter-fabric/velocity-react/blob/614b3b2f7687888e9cf418351c243a7792178d6d/velocity-component.js#L154

I wonder if there's a better way to detect `jsdom`.
Maybe we could use something like https://github.com/tmpvar/jsdom/issues/1537#issuecomment-229405327 instead?

**Comment by finneganh** [comment]
Looks like this is an issue with the new garbage collection, as it uses a Velocity API that's not shimmed out in server / test mode. Will get a new release out presently.

**Comment by moretti** [comment]
Thanks.

What do you think about using a different approach for detecting Node.js?
`process.env.NODE_ENV` is commonly used by webpack (`DefinePlugin`) and browserify (`envify`).

How about [`detect-node`](https://github.com/iliakan/detect-node)?

**Comment by moretti** [comment]
Oh, sorry never mind, I didn't realize that a PR has already been merged 😄 
https://github.com/twitter-fabric/velocity-react/pull/130

**Comment by Tomyail** [comment]
Thanks. good way to detect node

**Comment by finneganh** [comment]
Should be fixed in 1.1.7.

**Comment by moretti** [comment]
Thanks 👍
